libs/execution/src/lib/types/io-types/filesystem.ts

```diff
@@ -13,19 +13,19 @@ import { IOTypeImplementation } from './io-type-implementation';
  */
 export interface FileSystem extends IOTypeImplementation<IOType.FILE_SYSTEM> {
   /**
-   * Retrieves a file from the file system.
+   * Retrieves a file from the file system. Depending on the implementation relative Paths (./) and parent directories (../) are treated differntly.
```

Comment:
Why might `.` and `..` be treated differently in different implementations? I think they should have fixed semantics (i.e. `.` for current folder, `..` for parent folder), regardless of the concrete implementation.

Reply:
Right, i changed the docs